docs/specs/bug_and_feedback_submission.md

```diff
@@ -48,9 +48,40 @@ When putting an object into a bucket [the client must give it a key](https://doc
 
 This opens a possible attack vector for incurring large bills to SatoshiLabs by an attacker uploading large amounts of data to the buckets. This can't be 100% prevented as the API must remain open because Suite's users don't authenticate in any way. A rate limiting could prevent this but that would require some kind of application layer in between the clients and the S3. Instead, for start, we'll start by setting up a [CloudWatch alarm](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/AlarmThatSendsEmail.html) that will warn us if daily uploads exceed some threshold.
 
+## Data
+This section describes what data are sent in the query string. We start with smaller set of fields that can all fit into one level of nesting so that we don't have to decide [which encoding convention](https://stackoverflow.com/a/9547490/5698865) to use for nested fields. More 
+
+### Bugs
+- `description` The textual description of the bug written by the user.
+- `path` Current url path. Corresponds to the "category" in the Designs.
+- `screenshot` (optional) UUID of linked image.
+
+### Feedback
+- `rating` 1-5, 1 is best, 5 is worst.
+- `description` (optional) User's impressions of the app.
+- `screenshot` (optional) UUID of linked image.
```

Comment:
I know it is in design but does it make sense to attach a screenshot to a _Suggestion_? Could you discuss this with @matejzak?

Reply:
It's not the core functionality of the feedback part, but we'd still like to support it: The user can indicate in a simple drag&drop way what they are talking about (their specific use case).

Reply:
Well okay then :)